language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/constants/CommandConstants.java

```diff
@@ -153,4 +153,7 @@ public class CommandConstants {
     public static final String MAKE_CONSTRUCT_PUBLIC = "Convert '%s' to public";
     
     public static final String EXTRACT_TYPE = "Extract type";
+
+    public static final String MAKE_ANNOT_DECL_CONST = "Convert '%s' to constant";
```

Comment:
Is this the most suitable commandTitle since the keyword name is `const`? WDYT? @IMS94 @TharushiJay